Apps/Sandcastle/gallery/Drawing on Terrain.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Draw lines and polygons on terrain with mouse clicks.">
+    <meta name="cesium-sandcastle-labels" content="Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+
+    #viewChanged, #cameraChanged {
+        display: none;
+        background-color: red;
+        color: white;
+    }
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer',  {
+    selectionIndicator : false,
+    terrainProvider : Cesium.createWorldTerrain()
+});
+// Create a function to place a point given a world position.
```

Comment:
We try to avoid cluttering up sandcastle examples with inline comments unless there is something truly non-obvious.  I think most if not all of these comments can be removed.

Reply:
Good point - I just noticed most of the other Sandcastle examples don't have comments. I was treating it more as a tutorial, but perhaps I could write up an actual tutorial for that later.

Reply:
Yep, a tutorial would be the way to go with all that extra information